src/ch/imedias/rsccfx/localization/Bundle_de.properties

```diff
@@ -89,8 +89,8 @@ expertKeyserverIpLbl=Schl\u00fcsselserver IP-Addresse
 expertKeyserverHttpLbl=Schl\u00fcsselserver HTTP-Port
 expertVncPortLbl=VNC Port
 expertIcePortLbl=ICE Port
-expertUdpPackageSizeLbl=UDP Paketgr\u00F6sse
-expertForwardingPortLbl=lokaler Weiterleitungsport
+expertUdpPackageSizeLbl=UDP Paketgr\u00F6sse (Bytes)
+expertForwardingPortLbl=Lokaler Weiterleitungsport
```

Comment:
Maybe we need to find a better way what this exactly is.. Lokaler Weiterleitungsport doesnt sound to good... Also in English localForwardingPort.. Maybe proxyPort?